python/sqlflow_submitter/tensorflow/evaluate.py

```diff
@@ -76,7 +76,7 @@ def evaluate(datasource,
     model_params.update(feature_columns)
     if is_estimator:
         if is_pai:
-            define_tf_flags()
+            # define_tf_flags()
```

Comment:
Remove this line or add a TODO?

Reply:
Deleted this line, flags will be defined in the PAI code generator.